frontend/server.js

```diff
@@ -12,12 +13,18 @@ app.get("/", (req, res) => {
    res.sendFile(__dirname + '/index.html');
 });
 
-const httpsOptions = {
-    key: fs.readFileSync('./ssl/device.key'),
-    cert: fs.readFileSync('./ssl/localhost.crt')
-}
+if(HTTPS) {
 
-const server = https.createServer(httpsOptions, app).listen(port, () => {
-    console.log(`Frontend Server listening at ${port}`);
-});
+    const httpsOptions = {
```

Comment:
@fionnfuchs HTTPS is not necessary on Chrome – different to what I had originally thought.